common/src/main/java/com/wynntils/features/user/overlays/map/MiniMapOverlayFeature.java

```diff
@@ -50,6 +53,9 @@ public static class MiniMapOverlay extends Overlay {
         @Config
         public boolean renderUsingLinear = true;
 
+        @Config
+        public CustomColor cursorColor = new CustomColor(1f, 1f, 1f, 1f);
```

Comment:
Missing I18n. (If you delete your run configs in `.idea/runConfigurations` and reload the gradle project, you will have assertions enabled in dev env, not allowing missing I18n)